CHANGES.rst

```diff
@@ -1,23 +1,91 @@
 Change Log
 ==========
 
-v0.5.0
-------
+v0.5.0.stable.0
```

Comment:
We don't do this `.stable` business and it's not in the instructions, why change?

Reply:
I don't quite understand how releases are tagged and how the distinction between stable and dev is made...
Changed.

Reply:
Stable and releases are synonymous, and are just `X.Y.Z`. Dev is `X.Y.Z.dev0`

Reply:
Is tagging done via `__version__ ` in the `__init__.py` file? And I assume stable is differentiated from dev in the CircleCI config file via the line:

```
            tags:
              only: /^Release.*$/
```

But I don't see what this regex is looking for...?

Reply:
> Is tagging done via __version__ in the __init__.py file?

No, it's done via `git tag`, but this is done automatically when you do tell GitHub to make a release. 

> I don't see what this regex is looking for...?

Good call -- it was meant to look for the title standard we've been following when doing a GitHub release:

https://github.com/sphinx-gallery/sphinx-gallery/releases

But it looks like the tags actually land as `vX.X.X` so the regex is incorrect. Can you fix it here? I think `^v\d+\.\d+\.\d+$` might do it?

Reply:
> But it looks like the tags actually land as `vX.X.X` so the regex is incorrect. Can you fix it here? I think `^v\d+\.\d+\.\d+$` might do it?

That looks correct to me. Thanks, it all makes sense now.

Reply:
Excellent, I'll merge once you push a commit to fix this (or I can do it) and CIs are happy

Reply:
Last question, what do the backslashed (e.g. `/^Release.*$/`) around the regex do?

Reply:
Not sure, but they are always in the CircleCI config

https://circleci.com/docs/2.0/workflows/#executing-workflows-for-a-git-tag

Reply:
Thanks!